shopsafe-backend/src/main/java/com/google/sps/servlets/StoresServlet.java

```diff
@@ -62,9 +104,60 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
         // Todo: Get score of each store
 
-        // Todo: Return results
+        // Todo: Return stores with scores
+        Gson gson = new Gson();
+        response.setContentType("application/json;");
+        response.getWriter().println(gson.toJson(new Result(stores)));
+    }
 
-        response.setContentType("text/html");
-        response.getWriter().println(county.getCountyName());
+    /**
+     * Returns a list of Stores without scores.
+     */
+    public List<StoreNoScore> getStores(LatLng location) {
+
+        // List of stores that will be returned, it will be empty if there is an exception.
+        List<StoreNoScore> stores = new ArrayList<>();
+        
+        try {
+
+            // Read response of call to FCC API given lat and lng.
+            URL url = new URL(PLACE_URL + location.getLatitude() + "," + location.getLongitude()+ PLACE_TYPE + PLACE_KEY);
+            BufferedReader reader = new BufferedReader(new InputStreamReader(url.openStream()));
+            
+            // Store response in json, by reading each line.
+            StringBuffer json = new StringBuffer();
```

Comment:
Is there a reason why you chose StringBuffer over StringBuilder? I'm just wondering haha

Reply:
Not really, they do the same thing, but apparently StringBuilder is slightly faster. So, I made the switch.